src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,49 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.serviceimpl.CsvToFruitTransactionConverter;
```

Comment:
The `serviceimpl` classes should be in the `core.basesyntax.service.impl` package.